src/Tizen.Applications.Common/Tizen.Applications/TimeZoneChangedEventArgs.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// The class for the argument of the TimeZoneChanged EventHandler.
+    /// </summary>
+    /// <since_tizen> 11 </since_tizen>
+    public class TimeZoneChangedEventArgs : EventArgs
+    {
+
+        /// <summary>
+        /// Initializes TimeZoneChangedEventArgs class
+        /// </summary>
+        /// <param name="timeZone">The information of the Locale</param>
```

Comment:
missing `<param` for the second parameter, I suspect that it would be ok if the constructor was not public

Reply:
Done